### ISSUE TITLE
Error while creating map's Point field in Directus

### ISSUE THREAD

**Issue by satputekuldip** [directus/directus#22691]
### Describe the Bug

# Description:
I encountered an error when attempting to create a map's Point field in Directus. Below is the error message:

```json
{
  "errors": [
    {
      "message": {
        "length": 95,
        "name": "error",
        "severity": "ERROR",
        "code": "42704",
        "position": "48",
        "file": "parse_type.c",
        "line": "270",
        "routine": "typenameType"
      },
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}
```

### To Reproduce

1. Go to the Directus interface.
2. Attempt to create a new field of type 'Point' in a map.
3. The above error message appears.

### Directus Version

10.12.0

### Hosting Strategy

Self-Hosted (Docker Image)

### Database

PostgreSQL 16

**Comment by satputekuldip** [comment]
### This can be solved by Enabling PostGIS

If you are running PostgreSQL as a service from a cloud provider (not installing it yourself) then you probably already have PostGIS installed, and you just need to enable it in your database. Connect to your database as the postgres user or another super-user account, and run:

```sql
CREATE EXTENSION postgis;
```
Congratulations! PostGIS is now enabled.

Explore Your Install
You can find out what version you have, with [postgis_full_version()](https://postgis.net/docs/PostGIS_Full_Version.html).
```sql
SELECT PostGIS_Full_Version();
```
You can also check to see what alternate PostGIS versions might be installed.

```sql
SELECT * FROM pg_available_extensions WHERE name = 'postgis';
```

**Comment by divStar** [comment]
Perhaps for those, who **self-host** Directus 10+ and end up wanting to use the Map widget, but receiving the same error mentioned above: if you're **self-hosting using Docker** and `docker-compose`, you can just **replace** your [`postgres`](https://hub.docker.com/_/postgres) image with a corresponding [`postgis`](https://registry.hub.docker.com/r/postgis/postgis/) image.

> [!CAUTION]  
> Remember to **_backup your database volumes_**.

Use the corresponding version, e.g. if you use `PostgreSQL 16` (`postgres:latest` or `postgres:16` or similar), make sure to use `PostGIS 3.4.x with PostgreSQL 16` (`postgis/postgis:16-3.4`). It should be a drop-in replacement. You'll still want to activate the extension using the following statement:

```sql
CREATE EXTENSION postgis;
```
The output should say something about the extension having been successfully created.

Then you can use the following query to check whether PostGIS is available:
```sql
SELECT PostGIS_Version();
```
This should return a proper result stating what features are turned on etc.

You do **not** have to restart Directus itself, but you can refresh your current session. Now you should be able to create the Map widget / column.

**Tested** using Directus `10.13.1` and `postgis/postgis:16-3.4`.

**Comment by m4rrc0** [comment]
For reference, running directus and postgres **outside of Docker**, I had to do the following:

1. Install PostGIS on my system (Debian):
`sudo apt install postgis`

2. Connect to Directus DB:
`sudo -u postgres psql`
`\c your_directus_db_name`

3. Install PostGIS on the DB:
`CREATE EXTENSION postgis;`

4. Optional: Restart your process manager (if running one). For example:
`pm2 restart directus`

I know very little about Postgres and PostGIS so I hope this is enough. So far, so good...
It seems overly simple now but I had a few hiccups along the way so I figured I might as well share this process since it is not in the docs.
